mlrun/config.py

```diff
@@ -532,8 +533,54 @@
         },
     },
     "model_endpoint_monitoring": {
-        "serving_stream_args": {"shard_count": 1, "retention_period_hours": 24},
-        "application_stream_args": {"shard_count": 1, "retention_period_hours": 24},
+        "serving_stream": {
+            "v3io": {
+                "shard_count": 2,
+                "retention_period_hours": 24,
+                "num_workers": 1,
+                "min_replicas": 2,
+                "max_replicas": 2,
+            },
+            "kafka": {
```

Comment:
Can we also add a retention  config for `kafka`?

Reply:
Will leave it out for now because the old client we use doesn't expose a clear API for this, and this PR is already in the pipeline for too long, mostly due to 1.7.0 tasks preempting it.

Reply:
Added a TODO as discussed.